### DISCUSSION TITLE
Question: Mongodb+srv

### DISCUSSION THREAD

**Discussion by thecontinium** [michaelklishin/monger#228] [General]
Hi

Is the mongodb+Srv protocol supported. 

Cheers

**Comment by IliasDeros** [comment]
Yes, you can use it like so:

```clojure
(ns my-namespace (:require [monger.core :as monger]))
(defonce db (monger/connect-via-uri "mongodb+srv://user:pass@host/db"))
```

**Comment by cardoso-neto** [comment]
I'm getting a `A TXT record is only permitted to contain the keys [authsource, replicaset], but the TXT record for 'devopsai-ondemand.fcwir.mongodb.net' contains the keys [loadbalanced, authsource]` error when attempting to connect to an Atlas serverless cluster.
I'm using Metabase and Metabase seems to be using this driver: https://github.com/metabase/metabase/blob/b48e3077a15ec00d0b65f72079e5c2ebdf7a901a/modules/drivers/mongo/deps.edn

Are you guys sure you fully support the SRV DNS record?

**Reply by michaelklishin** [reply]
Monger does not implement any protocols and does not directly deal with any DNS records. The underlying MongoDB Java client does. Maybe it needs an update.
